### PR TITLE
Fix doccumentation issues

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/authentication.rst [hunk 1]
```diff
@@ -158,23 +156,23 @@ Some service of GeoMapFish has some host restriction if you mix the domain.
 Application authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To be considered as authenticated we should have the correct `Cookie` header,
-we also check the `Referrer` header to be sure that the user is coming from the same domain.
-If he is equals to the `Host` header, we consider that the user is coming from the same domain.
+To be considered as authenticated we should have the correct ``Cookie`` header,
+we also check the ``Referrer`` header to be sure that the user is coming from the same domain.
```
Comment (arnaud-morvan) [inline doc/integrator/authentication.rst:160]:
```suggestion
we also check the ``Referer`` header to be sure that the user is coming from the same domain.
```

**Review by arnaud-morvan — APPROVED** [review]

File: doc/import_ngeo_config.py [hunk 2]
```diff
@@ -21,7 +21,9 @@ def _format_type(type_definition: dict[str, Any]) -> str:
         if "typeArguments" in type_definition:
             postfix = f'\\<{", ".join([_format_type(t) for t in type_definition["typeArguments"]])}>'
 
-        return f":ref:`integrator_guide_ngeo_properties_{type_definition['name']}`{postfix}"
+        if type_definition["name"].lower() not in ():
```
Comment (ochriste) [inline doc/import_ngeo_config.py:24]: is this really ever going to do something ?

File: doc/integrator/authentication.rst (hunk 1 above)
Comment (ochriste) [reply doc/integrator/authentication.rst:160]: if this change is the result of automatic orthographic correction, cant we add a global rule to ignore that particular word ?

Comment (sbrunner) [reply doc/integrator/authentication.rst:160]: +1 :-)

File: doc/import_ngeo_config.py (hunk 2 above)
Comment (sbrunner) [reply doc/import_ngeo_config.py:24]: Oups remaining :-), Thanks

**Pull request merged**